### PR TITLE
Feature: flexible plot dates

### PR DESCRIPTION
**Pull request by laestrada**
This PR addresses issue #115. It contains:
- the addition of `bmk_start` and `bmk_end` config variables which allow for specifying any simulation time frame in the benchmark plots eg. 1Hr, 1Day, 1.5Days, etc
- Runs all plotting under a single script `./run_benchmark.py <config_file_name>`, which based on the config file, either runs the default plotting benchmark submodule (based on 1mo script) or the 1yr_fullchem/ 1yr_tt benchmarking submodules
- removes `prior_to_13` config variable as this feature is now deprecated

### CODE REVIEW

File: benchmark/modules/run_1yr_fullchem_benchmark.py
Comment (lizziel) [inline benchmark/modules/run_1yr_fullchem_benchmark.py:69]: Do we need to supply default values for year here if the yaml file will always have start/end? I'm thinking ahead to when we change year since we'll have to remember to update these values, which might not be obvious since they are set in the config file.

File: benchmark/run_benchmark.py
Comment (lizziel) [inline benchmark/run_benchmark.py:420]: Is the header above this section ("Jul2016") still accurate? It looks like the new string will be a date range rather than month/year. Same comment for the other sections that have this.

**Review by lizziel — CHANGES_REQUESTED** [review]
This looks good! I added a few minor comments on specific lines. I also wonder if there could be text somewhere, maybe in the yaml files, that outlines the constaints of setting the start and end dates. For example, you should be able to use the 1-month benchmark config for any duration given ref and dev are the same ranges, not just 1-month. But I assume you can't do that for the 1-year. And are there limitations on if the ref and dev durations are different (e.g. comparing two months that have different number of days)? Would this be better suited for ReadTheDocs?

File: benchmark/run_benchmark.py
Comment (laestrada) [reply benchmark/run_benchmark.py:321]: I believe we agreed we can remove the is_legacy option once 14.0 is released. I will make a separate github issue for it though

File: benchmark/modules/run_1yr_fullchem_benchmark.py
Comment (laestrada) [reply benchmark/modules/run_1yr_fullchem_benchmark.py:69]: Good point, I was initially trying to make sure run_1yr_fullchem_benchmark.py was functional as a standalone script, but I think that just adds unnecessary confusion. Removed

File: benchmark/run_benchmark.py
Comment (laestrada) [reply benchmark/run_benchmark.py:420]: Good catch -- fixed!

**Comment by laestrada** [comment]
@lizziel Thanks for the review -- I have updated the PR accordingly! I will wait for a final approval before merge.

File: benchmark/run_benchmark.py
Comment (lizziel) [reply benchmark/run_benchmark.py:321]: Sounds good.

**Review by lizziel — APPROVED** [review]

**Pull request merged**
